blocks/columns/columns.css

```diff
@@ -136,3 +136,44 @@
     font-size: 22px;
   }
 }
+
+/***************************************************************
+* tab like columns
+***************************************************************/
+.columns.tabs {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: end;
+  border-bottom: 1px solid var(--highlight-background-color);
+  overflow: auto;
+}
+
+.columns.tabs>div {
```

Comment:
can we cleanup this css and make it concise.

combine classes in a single line if possible